s30278-GrzegorzPiorek/3. Procesy i strumienie/Zadania_1.txt

```diff
@@ -0,0 +1,498 @@
+www-data 4035209  0.0  0.6 305244 27904 ?        S    00:00   0:00 /usr/sbin/apa
+www-data 4035210  0.0  0.6 302072 25932 ?        S    00:00   0:00 /usr/sbin/apa
+www-data 4035211  0.0  0.5 227724 22232 ?        S    00:00   0:00 /usr/sbin/apa
+www-data 4035212  0.0  0.5 227724 21468 ?        S    00:00   0:00 /usr/sbin/apa
+www-data 4035213  0.0  0.8 236864 32768 ?        S    00:00   0:00 /usr/sbin/apa
+www-data 4035244  0.0  0.6 302080 26216 ?        S    00:08   0:00 /usr/sbin/apa
+www-data 4035412  0.0  0.5 227548 22132 ?        S    00:19   0:00 /usr/sbin/apa
+root     4040803  0.0  0.0      0     0 ?        I    00:29   0:15 [kworker/0:2-
+root     4051296  0.0  0.3  58152 14884 ?        Ss   11:00   0:00 sshd: s30278 
+s30278   4051299  0.0  0.2  15424  8672 ?        Ss   11:00   0:00 /lib/systemd/
+s30278   4051300  0.0  0.1 214896  4168 ?        S    11:00   0:00 (sd-pam)
+s30278   4051317  0.0  0.1  58284  6916 ?        S    11:00   0:00 sshd: s30278@
+s30278   4051318  0.0  0.1   8412  5080 pts/14   Ss   11:00   0:00 -bash
+root     4051365  0.0  0.0      0     0 ?        I    11:05   0:00 [kworker/u2:3
+www-data 4051374  0.0  0.4 227276 16332 ?        S    11:06   0:00 /usr/sbin/apa
+www-data 4051375  0.0  0.4 227276 16344 ?        S    11:06   0:00 /usr/sbin/apa
+www-data 4051376  0.0  0.4 227276 16360 ?        S    11:06   0:00 /usr/sbin/apa
+root     4051483  0.0  0.0      0     0 ?        I    11:17   0:00 [kworker/u2:2
+root     4052056  0.0  0.0      0     0 ?        I    11:38   0:00 [kworker/0:0-
+root     4052243  0.0  0.0      0     0 ?        I    11:43   0:00 [kworker/0:1-
+root     4052306  0.0  0.0      0     0 ?        I    11:46   0:00 [kworker/0:3-
+root     4052379  0.0  0.0      0     0 ?        I    11:47   0:00 [kworker/u2:0
+root     4052430  0.0  0.3  58152 14908 ?        Ss   11:50   0:00 sshd: s32410 
+s32410   4052435  0.3  0.2  15304  8184 ?        Ss   11:50   0:00 /lib/systemd/
+s32410   4052436  0.0  0.1 214896  4168 ?        S    11:50   0:00 (sd-pam)
+s32410   4052455  0.0  0.1  58152  7044 ?        S    11:50   0:00 sshd: s32410@
+s32410   4052456  0.0  0.1   8412  5116 pts/15   Ss+  11:50   0:00 -bash
+s30278   4052465  0.0  0.0  10084  3592 pts/14   R+   11:50   0:00 ps axu
+"
+
+-Polecenie "ps axu" zawiera 3 znaki które punktują konkretne procesy oraz informacje o nich. 
+-Polecenie pokazuje wszystkie procesy zależne oraz niezależne od konsoli, również procesy użytkownika "root", pokazuje równiez dzięki znakowi "u" dodatkowe informacje o każdym wypisanym procesie.
+
+Jak myślisz, co oznacza znak zapytania w kolumnie 2?
+Nie wiesz? A gdzie może być wyjaśnienie?
+
+kolumna TTY oznacza terminal do którego dane procesy są przypisane, jeśli występują w tej kolumnie znaki zapytania, oznacza to, że dany proces nie jest przypisany do konkretnej sesji terminala i działa niezależnie od niego. 
+
+B: Jak wykonasz poniższe 2 zadania?
+
+1. Wyświetl wszystkie procesy bash
+
+-"ps axu | grep [b]ash"
```

Comment:
To wyświetla tylko wiersze zawierające słowo `bash` a nie procesy o nazwie bash. Poszukać w `man ps` jak to zrobić poprawnie.